src/coreComponents/constitutive/permeability/ParallelPlatesPermeability.hpp

```diff
@@ -0,0 +1,125 @@
+/*
+ * ------------------------------------------------------------------------------------------------------------
+ * SPDX-License-Identifier: LGPL-2.1-only
+ *
+ * Copyright (c) 2018-2020 Lawrence Livermore National Security LLC
+ * Copyright (c) 2018-2020 The Board of Trustees of the Leland Stanford Junior University
+ * Copyright (c) 2018-2020 Total, S.A
+ * Copyright (c) 2019-     GEOSX Contributors
+ * All rights reserved
+ *
+ * See top level LICENSE, COPYRIGHT, CONTRIBUTORS, NOTICE, and ACKNOWLEDGEMENTS files for details.
+ * ------------------------------------------------------------------------------------------------------------
+ */
+
+/**
+ * @file ParallelPlatesPermeability.hpp
+ */
+
+#ifndef GEOSX_CONSTITUTIVE_PERMEABILITY_PARALLELPLATESPERMEABILITY_HPP_
+#define GEOSX_CONSTITUTIVE_PERMEABILITY_PARALLELPLATESPERMEABILITY_HPP_
+
+#include "constitutive/permeability/PermeabilityBase.hpp"
+
+
+namespace geosx
+{
+namespace constitutive
+{
+
+class ParallelPlatesPermeabilityUpdate : public PermeabilityBaseUpdate
```

Comment:
I propose to call it "conductivity" or "fluid conductivity" instead of permeability.

Reply:
I would keep this as is. It is a permeability model after all and it will be stored as permeability. It just makes it a lot easier to handle if it's the same field as for the other regions. I do not like that we are computing it as aperture^3 instead of aperture^2 but it makes it a lot more convenient to use it the it has to be used.